services/user.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str | None = None,
+        first_name: str = "",
+        last_name: str = "",
+) -> None:
+    get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
```

Comment:
This is possible approach, but to be safe it is better to write optional parameters to DB only when they are actually being passed. For example here you fill out first name and last name with empty strings. But what if some other part of our application expects those fields to be NULL since we know we didn't send any names there. It will lead to your DB containing some garbage values such as empty strings which may in turn affect other queries where we filter by NULL values. So in this scenario it will be better to check if param is present before you write to db